packages/calypso-products/src/experiments.ts

```diff
@@ -1,24 +0,0 @@
-type Experiment = { [ key: string ]: string };
-type Experiments = { [ key: string ]: Experiment };
-
-const experiments: Experiments = {};
-const PLANS_LIST_NAMESPACE = 'plans-list';
-
-const setExperiment = ( namespace: string, experimentName: string, variation: string ): void => {
-	if ( ! experiments[ namespace ] ) {
-		experiments[ namespace ] = {};
-	}
-	experiments[ namespace ][ experimentName ] = variation;
-};
-
-const getExperiment = ( namespace: string, experimentName: string ): string | undefined => {
-	return experiments[ namespace ]?.[ experimentName ];
-};
-
-export const setPlansListExperiment = ( experimentName: string, variation: string ): void => {
```

Comment:
Can we confirm these aren't used as part of the package somewhere externally?

Reply:
I couldn't find any matches on `opengrok`, but it would be great if other folks double-checked too

Reply:
Didn't spot any usages either 👍